packages/oc-template-handlebars-compiler/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "oc-template-handlebars-compiler",
+  "version": "1.0.0",
+  "description": "OC-Template-Handlebars-Compiler",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/opencomponents/oc-template-handlebars.git"
+  },
+  "keywords": [],
+  "author": "Nick Balestra <nick@balestra.ch>",
```

Comment:
Yes, npm support few formats for some stuff, but actually, now that you bring it up, I think I agree with you, perhaps makes it a little be easier to parse in case you ever need.